src/singlefit.jl

```diff
@@ -473,6 +473,7 @@ function fit_binned_trunc_gauss(h_nocut::Histogram, cuts::NamedTuple{(:low, :hig
     report = (
             f_fit = x -> Base.Fix2(f_fit, v_ml)(x) * bin_width,
             h = h_nocut,
+            h_cut = h,
```

Comment:
There is a recipe for this, did you change that too so that it still produces a plot?

Reply:
I updated the recipe